jenkins/spark-tests.sh

```diff
@@ -99,6 +99,12 @@ tar xzf "$RAPIDS_INT_TESTS_TGZ" -C $ARTF_ROOT && rm -f "$RAPIDS_INT_TESTS_TGZ"
 $MVN_GET_CMD -DremoteRepositories=$SPARK_REPO \
     -DgroupId=org.apache -DartifactId=spark -Dversion=$SPARK_VER -Dclassifier=bin-hadoop3.2 -Dpackaging=tgz
 
+# Donwload parquet-hadoop jar for parquet-read encryption tests
+PARQUET_HADOOP_VER=`mvn help:evaluate -q -N -Dexpression=parquet.hadoop.version -DforceStdout -Dbuildver=${SHUFFLE_SPARK_SHIM/spark/}`
+[[ "$(printf '%s\n' "1.12.0" "$PARQUET_HADOOP_VER" | sort -V | head -n1)" = "1.12.0" ]] && \
```

Comment:
Is it necessary to use `sort` and `head` here?  How could more than one version be emitted?

Reply:
Is it necessary to use `sort` and `head` here?  --> There are other ways to compare strings, while `sort` `head` is also a simple way.

How could more than one version be emitted? ---> `parquet.hadoop.version` version is set in the pom.xml, there is only one version for every spark-shim.  https://github.com/NVIDIA/spark-rapids/blob/branch-22.08/pom.xml#L221